frontend/pages/hosts/details/HostDetailsPage/_styles.scss

```diff
@@ -358,12 +358,6 @@
     text-decoration: none;
   }
 
-  #back-chevron {
-    width: 16px;
-    height: 16px;
-    margin-right: $pad-small;
-  }
```

Comment:
There is a `.button img` class in this stylesheet that applies `transform: scale(0.5)`. That was resizing the icon down to 8 x 8 instead of 16 x 16. Because the icon is 32px, we can rely on the general transform to resize it.

This is an example of our need to standardize styling. On some pages we apply this transform, and others we don't.